apps/teamsjs-diagnostic-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Specific reason to choose `es5`?

Reply:
Another autogenerated file! But from what I understand, it's a good option because it works well on most modern browsers and systems, making sure it runs smoothly even on older browsers that can't handle the latest JavaScript updates.